### DISCUSSION TITLE
How to copy frames?

### DISCUSSION THREAD

**Discussion by VivaLaVaati** [keiwando/toonsquid#327] [General]
I’m new to toonsquid. How do I copy the frame/drawing I drew to put it later on in the timeline to loop an animation? I’m trying to do a run cycle for a bigger animation and eventually the frames are gonna have to  loop so I want to know how to do it.

**Comment by Cat-Butler** [comment]
If you are animating by drawing frame-by-frame, tap on a frame in the timeline to pop up a menu including copy and paste.

If you are animating using keyframes, make sure you turn off pixel edit or path edit mode (Transform tool, icons at top right), turn on keyframe mode (little diamond icon under the timeline) then tap on any keyframe icon (same diamond icons, but in the timeline channels for the selected object) to pop up a similar menu.
